.travis.yml

```diff
@@ -6,5 +6,5 @@ install:
  - go get github.com/fzipp/gocyclo
  - go get github.com/client9/misspell/...
  - go get github.com/gordonklaus/ineffassign
- - go get gopkg.in/Shopify/sarama.v1
+ - go get -d ...  # Downloads all the dependencies of naffka
```

Comment:
does this make the others redundant, or?

Reply:
I think all the others are required as they're the linters, so don't get imported by anything in the naffka package.